docs/awesome-pets/papers/applications/set/psi.md

```diff
@@ -53,13 +53,18 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
   *Gayathri Garimella, Payman Mohassel, Mike Rosulek, Saeed Sadeghian, Jaspal Singh*  
   PKC 2021, [eprint](https://eprint.iacr.org/2021/243), GMRS21
   
+- PSI from PaXoS: Fast, Malicious Private Set Intersection
+  *Benny Pinkas, Mike Rosulek, Ni Trieu, Avishay Yanai*
```

Comment:
Same here

Reply:
I modified it here(in my branch).
![image](https://github.com/secretflow/secretflow/assets/54618933/7501d7da-b905-456c-8fc9-506272cefd48)
Is that going to show up here?

Reply:
> Is that going to show up here?

LGTM, I'm ok with its position